src/components/ExamScheduleViewer.tsx

```
@@ -109,6 +129,16 @@
         <div className="mt-5 text-center text-secondary-text italic text-base select-none">
           Gefundene Einträge: {filteredEntries.length} / {entries.length}
         </div>
+        <div className="flex justify-center">
+          <button
+            className="cursor-pointer border rounded p-1"
+            onClick={() => {
+              download("pruefungsplan.ics", generateICSFile(filteredEntries));
+            }}
+          >
+            Gefilterte Prüfungen als ics Datei exportieren (Kalenderdatei)
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)